backend/tests/controllers/test_executions/test_start_test.py

```diff
@@ -120,19 +145,9 @@ def test_creates_all_data_models(db_session: Session, execute: Execute):
 
 def test_invalid_artefact_format(execute: Execute):
     """Artefact with invalid format no store should not be created"""
-    response = execute(
-        {
-            "family": "snap",
-            "name": "core22",
-            "version": "abec123",
-            "revision": 123,
-            "track": "22",
-            "arch": "arm64",
-            "execution_stage": "beta",
-            "environment": "cm3",
-            "ci_link": "http://localhost",
-        },
-    )
+    request = test_request.copy()
+    del request["store"]
+    response = execute(request)
```

Comment:
This test seems to be testing the same behaviour as the newly added `test_required_fields`, do you think we should modify that one to check for `store` as well?

Reply:
Ended up splitting the request models into two. The benefit of that is the API schema and docs now reflect that depending on the family, different requirements are in place